extension/test/e2e/pageObjects/PaypalPopupPage.js

```diff
@@ -0,0 +1,20 @@
+export default class PaypalPopupPage {
+  constructor(page) {
+    this.page = page
+  }
+
+  async handlePaypalPopUp() {
+    await this.page.waitForSelector('#email')
+    await this.page.type('#email', 'sb-urfdg23058190@personal.example.com')
```

Comment:
is it random input? Do we need to use specific testing data provided by Paypal?

Reply:
It's a generated email for testing, but not random.